LiveChat/components/Chat.js

```diff
@@ -164,6 +75,11 @@ Chat.propTypes = {
   isChatOn: PropTypes.bool.isRequired,
   greeting: PropTypes.string.isRequired,
   noAgents: PropTypes.string.isRequired,
+  messages: PropTypes.array.isRequired,
+  customer: PropTypes.object,
+  onInputChange: PropTypes.func.isRequired,
+  isTyping: PropTypes.bool.isRequired,
+  connectionState: PropTypes.string.isRequired,
```

Comment:
IMHO the `connectionState` should have a `PropTypes .oneOf` type with listed only possible values. After all, it shouldn't be just any string.